Update 08.md (#18)
dotnet-foundation/newsletter: 2019/08.md

Tickets are available at https://www.cloudbrew.be.

## .NET Foundation Project Updates
### [Chemistry Add-In for Word (Chem4Word)](https://www.chem4word.co.uk/)

> It seems like it's all gone quiet on Chem4Word recently, but we've been working _extremely_ hard on addressing some licensing concerns.  _Chem4Word_ is currently released under the Apache 2.0 license, but its structure 'sketcher' is a 3rd-party product  which uses GPL.  

Reviewer: ```suggestion
It seems like it's all gone quiet on Chem4Word recently, but we've been working _extremely_ hard on addressing some licensing concerns.  _Chem4Word_ is currently released under the Apache 2.0 license, but its structure 'sketcher' is a 3rd-party product which uses GPL.  
```

> So we've been writing a new sketcher - the _Advanced CML-based Molecule Editor_ (or ACME for short).  It's written in C# and uses Windows Presentation Foundation for its graphics.  It can beused as a drop-in structure editor for any Windows Forms or WPF application.

Reviewer: ```suggestion
So we've been writing a new sketcher - the _Advanced CML-based Molecule Editor_ (or ACME for short).  It's written in C# and uses Windows Presentation Foundation for its graphics.  It can be used as a drop-in structure editor for any Windows Forms or WPF application.
```

 ![ACME in Action](https://user-images.githubusercontent.com/10074162/62897078-5f8fcd00-bd4a-11e9-8068-e153788e579f.png)

> Having our own editor also means we have complete control over what we can and can't edit in CML  This means we can add new features much more easily.  And keep our users happy!

Reviewer: ```suggestion
Having our own editor also means we have complete control over what we can and can't edit in CML.  This means we can add new features much more easily.  And keep our users happy!
```

Reviewer 2: Suggested change

It can be used as a drop-in structure editor for any Windows

I.e. add space between "can" and "be"
